modules/core/src/main/java/org/apache/ignite/internal/processors/rest/handlers/GridRestCommandHandlerAdapter.java

```diff
@@ -26,8 +26,6 @@
  * Abstract command handler.
  */
 public abstract class GridRestCommandHandlerAdapter implements GridRestCommandHandler {
-    /** Used cache name in case the name was not defined in a request. */
-    protected static final String DFLT_CACHE_NAME = "default";
```

Comment:
The default cacheName is safe fallback when cache name is not provided in the request. It is part of rest document.

https://apacheignite.readme.io/docs/rest-api#put

When request do not have cacheName since it is optional param then this DFLT_CACHE_NAME is used https://github.com/apache/ignite/pull/8041/files#diff-a3477d5e0cfdfcceed3371fc899a9d15L30

Reply:
I understand this, but this is pointless. It is not safe fallback, because there is no "default" cache. If you will try to use any API without a name you will receive the following error:

_org.apache.ignite.IgniteCheckedException: Failed to find cache for given cache name: default_

Better to replace it by the correctly formed error message that will point to the actual problem of the request. API document will be updated accordingly when PR will be merged.

Reply:
Can you please confirm in your configuration you have this example config file.

https://github.com/apache/ignite/blob/master/examples/config/example-cache.xml#L26-L32

This has instructions on default cache config.

My understanding is when node startup then the default cache should already be available.

Reply:
This config is just an example configuration. You don't have to use it. Obviously, it will be working, because cache is defined there. Real configs will not have such cache, but it still will be expected.